QUIC_WARNING_FLAGS not being passed down to quicperf

Because if they were, I would not see this:
```
Scanning dependencies of target quicperf
[ 73%] Building CXX object src/perf/bin/CMakeFiles/quicperf.dir/appmain.cpp.o
In file included from /vagrant/src/perf/bin/appmain.cpp:12:0:
/vagrant/src/perf/lib/PerfHelpers.h:241:41: warning: multi-character character constant [-Wmultichar]
         QuicPoolInitialize(Paged, Size, QUIC_POOL_PERF, &Pool);
                                         ^~~~~~~~~~~~~~
/vagrant/src/perf/lib/PerfHelpers.h:262:46: warning: multi-character character constant [-Wmultichar]
         QuicPoolInitialize(Paged, sizeof(T), QUIC_POOL_PERF, &Pool);
                                              ^~~~~~~~~~~~~~
```

--- a/src/perf/bin/CMakeLists.txt
+++ b/src/perf/bin/CMakeLists.txt
@@ -12,4 +12,4 @@ add_executable(quicperf ${SOURCES})
 
 set_property(TARGET quicperf PROPERTY FOLDER "perf")
 
-target_link_libraries(quicperf perflib msquic platform perfbin.clog)
+target_link_libraries(quicperf inc warnings perflib msquic platform perfbin.clog)

--- a/src/perf/bin/appmain.cpp
+++ b/src/perf/bin/appmain.cpp
@@ -100,7 +100,7 @@ QUIC_STATUS
 QuicKernelMain(
     _In_ int argc,
     _In_reads_(argc) _Null_terminated_ char* argv[],
-    _In_ bool KeyboardWait,
+    _In_ bool /*KeyboardWait*/,
     _In_ QUIC_SEC_CONFIG_PARAMS* SelfSignedParams
     ) {
     size_t TotalLength = sizeof(argc);
@@ -257,9 +257,9 @@ Exit:
 
 void
 DatapathReceiveUserMode(
-    _In_ QUIC_DATAPATH_BINDING* Binding,
+    _In_ QUIC_DATAPATH_BINDING*,
     _In_ void* Context,
-    _In_ QUIC_RECV_DATAGRAM* DatagramChain
+    _In_ QUIC_RECV_DATAGRAM*
     )
 {
     QUIC_EVENT* Event = static_cast<QUIC_EVENT*>(Context);
@@ -268,9 +268,9 @@ DatapathReceiveUserMode(
 
 void
 DatapathUnreachable(
-    _In_ QUIC_DATAPATH_BINDING* Binding,
-    _In_ void* Context,
-    _In_ const QUIC_ADDR* RemoteAddress
+    _In_ QUIC_DATAPATH_BINDING*,
+    _In_ void*,
+    _In_ const QUIC_ADDR*
     )
 {
     //

